src/combobox/ComboBox.ts

```diff
@@ -169,19 +162,19 @@ export default class ComboBox extends ComboBoxBase<ComboBoxProperties> {
 	private _onInputKeyDown(event: KeyboardEvent) {
 		const { results = [] } = this.properties;
 
-		switch (event.keyCode) {
-			case keys.up:
+		switch (event.which) {
```

Comment:
I thought it would be good to standardize use of `event.keyCode` or `event.which`, since we currently use both. Should we bother/should it be in a different PR?